unidist/core/backends/mpi/core/serialization.py

```diff
@@ -229,3 +229,67 @@ def deserialize(self, s_data):
         unpacked_data = msgpack.unpackb(s_data, object_hook=self._decode_custom)
         gc.enable()
         return unpacked_data
+
+
+class SimpleSerializer:
+    """
+    Class for simple data serialization/de-serialization for MPI comminication.
+
+    Notes
+    -----
+    Uses cloudpickle and pickle libraries as a separate APIs.
+    """
+
+    def serialize(self, data):
+        """
+        Encode with a cloudpickle library.
+
+        Parameters
+        ----------
+        obj : object
+            Python object.
+
+        Returns
+        -------
+        bytearray
+            Array of serialized bytes.
+        """
+        return cpkl.dumps(data)
+
+    def serialize_pickle(self, data):
+        """
+        Encode with a pickle library.
+
+        Parameters
+        ----------
+        obj : object
+            Python object.
+
+        Returns
+        -------
+        bytearray
+            Array of serialized bytes.
+        """
+        return pkl.dumps(data)
+
+    def deserialize(self, data):
+        """
+        De-serialization with cloudpickle library.
+
+        Parameters
+        ----------
+        obj : object
```

Comment:
Let's be more concrete in types:
```suggestion
        obj : bytearray
```